app/Filament/Resources/GroupResource.php

```diff
@@ -0,0 +1,234 @@
+	}
+	
+	public static function getNavigationBadge(): ?string
+	{
+		if (! request()->attributes->has('group')) {
+			return static::getModel()::count();
+		}
+		
+		return parent::getNavigationBadge();
+	}
+	
+	public static function getPluralModelLabel(): string
+	{
+		if ($group = request()->attributes->get('group')) {
+			return $group->name;
+		}
+		
+		return parent::getPluralModelLabel();
+	}
+	
+	public static function form(Form $form): Form
+	{
+		return $form
+			->schema([
+				Forms\Components\Fieldset::make('Group')->schema([
+					Forms\Components\TextInput::make('name')
+						->columnSpanFull()
+						->required()
+						->maxLength(255),
+					Forms\Components\Textarea::make('description')
+						->columnSpanFull()
+						->required(),
+					Forms\Components\TextInput::make('domain')
+						->required()
+						->maxLength(255)
+						->disabled(fn() => ! Auth::user()->isSuperAdmin())
+						->rules(['']),
+					Forms\Components\Select::make('domain_status')
+						->required()
+						->options(DomainStatus::class)
+						->default(DomainStatus::Pending)
+						->disabled(fn() => ! Auth::user()->isSuperAdmin()),
+					Forms\Components\Select::make('continent')
+						->required()
+						->options(Continent::class)
+						->default(Continent::NorthAmerica),
+					Forms\Components\TextInput::make('region')
+						->maxLength(255)
+						->helperText('eg. “New York” for NYC'),
+					Forms\Components\Select::make('status')
+						->required()
+						->options(GroupStatus::class)
+						->default(GroupStatus::Planned),
+					Forms\Components\TextInput::make('frequency')
+						->required()
+						->maxLength(255)
+						->default('bi-monthly'),
+					Forms\Components\Select::make('timezone')
+						->searchable()
+						->options(
+							collect(timezone_identifiers_list())
+								->mapWithKeys(fn(string $timezone) => [$timezone => str($timezone)->after('/')->replace('_', ' ')->toString()])
+								->groupBy(fn(string $timezone, string $key) => str($key)->before('/')->replace('_', ' ')->toString(), preserveKeys: true)
+						)
+						->required()
+						->default('America/New_York'),
+				]),
+				Forms\Components\Fieldset::make('Contact')->schema([
+					Forms\Components\TextInput::make('email')
+						->email()
+						->maxLength(255),
+				]),
+				Forms\Components\Fieldset::make('Links')->schema([
+					Forms\Components\TextInput::make('bsky_url')
+						->url()
+						->label('Bluesky')
+						->maxLength(255),
+					Forms\Components\TextInput::make('twitter_url')
+						->url()
+						->label('Twitter')
+						->maxLength(255)
+						->hidden(fn() => ! Auth::user()->isSuperAdmin()),
+					Forms\Components\TextInput::make('meetup_url')
+						->url()
+						->label('Meetup')
+						->maxLength(255),
+				]),
+				Forms\Components\Fieldset::make('Coordinates')->schema([
+					Forms\Components\TextInput::make('latitude')
+						->numeric()
+						->rules(['required', 'numeric', 'between:-90,90', 'decimal:2,8']),
```

Comment:
```suggestion
						->numeric()
						->required()
						->rules(['required', 'numeric', 'between:-90,90', 'decimal:2,8']),
```
We also want the required method here so it's consistent with the rules, this way the input field will show the red *.